scripts/build_and_deploy_custom_dc_autopush.sh

```diff
@@ -57,3 +59,6 @@ gcloud run deploy dc-autopush \
     --image gcr.io/datcom-ci/datacommons-website-compose:latest \
     --region us-central1 \
     --update-env-vars RESTART_TIMESTAMP="$(date)"
+
+# Run tests against dc-autopush
+./run_cdc_tests.sh
```

Comment:
this script is suppose to run after PR submit, which should have done the tests. So no need to add the test here.